omod/src/test/java/org/openmrs/module/fhir2/providers/FhirTaskFhirResourceProviderTest.java

```diff
@@ -29,7 +29,7 @@
 import org.openmrs.module.fhir2.api.FhirTaskService;
 
 @RunWith(MockitoJUnitRunner.class)
-public class TaskFhirResourceProviderTest {
+public class FhirTaskFhirResourceProviderTest {
```

Comment:
I prefer `TaskFhirResourceProviderTest` instead of `FhirTaskFhirResourceProviderTest`